### DISCUSSION TITLE
Add Codename Flanker Su-30 Mod

### DISCUSSION THREAD

**Discussion by Jaliosky** [Khopa/dcs_liberation#2921] [Q&A] [answered]
The Su-30 mod from codename flanker could be added in the future, as it is a good mod for the modern era.

**Comment by Starfire13** [comment, marked as answer]
Hi. Please add mod support requests under issues, not as Q&A.

**Reply by Jaliosky** [reply]
Thanks, I don't have much experience on the platform.

**Reply by Starfire13** [reply]
No worries. Just click on Issues, then Mod Support Request, and fill in the form.
